intertidal/elevation.py

```diff
@@ -1494,44 +1495,55 @@ def intertidal_cli(
     configure_s3_access(cloud_defaults=True, aws_unsigned=aws_unsigned)
 
     # Create output folder. If it doesn't exist, create it
-    output_dir = f"data/interim/{study_area}/{start_date}-{end_date}-spearman"
+    output_dir = f"data/interim/{study_area}/{start_date}-{end_date}"
     os.makedirs(output_dir, exist_ok=True)
 
     try:
         log.info(f"Study area {study_area}: Loading satellite data")
-
-        # Connect to datacube to load data
-        dc = datacube.Datacube(app="Intertidal_CLI")
-
+        
         # Create local dask cluster to improve data load time
         client = create_local_dask_cluster(return_client=True)
 
-        satellite_ds = load_data(
-            dc=dc,
-            study_area=study_area,
-            time_range=(start_date, end_date),
-            resolution=resolution,
-            crs="EPSG:3577",
-            include_s2=True,
-            include_ls=True,
-            filter_gqa=True,
-            max_cloudcover=90,
-            skip_broken_datasets=True,
-        )
+        if study_area == "testing":
```

Comment:
Use pickle file instead of loading data from datacube if study area == "testing"